DataFormats/PatCandidates/src/PackedCandidate.cc

```diff
@@ -105,6 +108,43 @@ void pat::PackedCandidate::unpackCovariance() const {
     unpackCovarianceElement(*m, packedCovariance_.dxydz, 3, 4);
     unpackCovarianceElement(*m, packedCovariance_.dlambdadz, 1, 4);
     unpackCovarianceElement(*m, packedCovariance_.dphidxy, 2, 3);
+
+    if (forcePosDef) {
+      //      std::cout<<"unpacking enforcing positive-definite cov matrix"<<std::endl;
+      //calculate the determinant and verify positivity
+      double det_(0);
+      bool computed_ = m->Det(det_);
```

Comment:
please follow CMSSW style (lower case camelCaps, no trailing underscore in locals); the comment is for the full file

Reply:
thanks for the reminder. "camelCaps" rule for local could be added to http://cms-sw.github.io/cms_coding_rules.html (unless I could not find it)